ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_pybind.cpp

```diff
@@ -120,6 +121,151 @@ void py_bind_conv2d(py::module& module) {
+        },
+        py::kw_only(),
+        py::arg("weight_tensor"),
+        py::arg("input_memory_config"),
+        py::arg("weights_format"),
+        py::arg("in_channels"),
+        py::arg("out_channels"),
+        py::arg("batch_size"),
+        py::arg("input_height"),
+        py::arg("input_width"),
+        py::arg("kernel_size"),
+        py::arg("stride"),
+        py::arg("padding"),
+        py::arg("dilation"),
+        py::arg("groups"),
+        py::arg("device"),
+        py::arg("conv_config") = std::nullopt);
+
+    module.def(
+        "prepare_conv_bias",
+        [](const ttnn::Tensor& bias_tensor,
+            const ttnn::MemoryConfig& input_memory_config,
+            uint32_t in_channels,
+            uint32_t out_channels,
+            uint32_t batch_size,
+            uint32_t input_height,
+            uint32_t input_width,
+            std::array<uint32_t, 2> kernel_size,
+            std::array<uint32_t, 2> stride,
+            std::array<uint32_t, 2> padding,
+            std::array<uint32_t, 2> dilation,
+            uint32_t groups,
+            ttnn::Device *device,
+            std::optional<const Conv2dConfig> conv_config_) -> ttnn::Tensor {
+                return ttnn::operations::conv::conv2d::prepare_conv_bias(
+                    bias_tensor, input_memory_config, in_channels, out_channels, batch_size, input_height, input_width, kernel_size, stride, padding, dilation, groups, device, conv_config_);
+        },
+        py::kw_only(),
+        py::arg("bias_tensor"),
+        py::arg("input_memory_config"),
+        py::arg("in_channels"),
+        py::arg("out_channels"),
+        py::arg("batch_size"),
+        py::arg("input_height"),
+        py::arg("input_width"),
+        py::arg("kernel_size"),
+        py::arg("stride"),
+        py::arg("padding"),
+        py::arg("dilation"),
+        py::arg("groups"),
+        py::arg("device"),
+        py::arg("conv_config") = std::nullopt);
+
+    module.def(
+        "prepare_conv_bias",
+        [](const ttnn::Tensor& bias_tensor,
+            const ttnn::MemoryConfig& input_memory_config,
+            uint32_t in_channels,
+            uint32_t out_channels,
+            uint32_t batch_size,
+            uint32_t input_height,
+            uint32_t input_width,
+            std::array<uint32_t, 2> kernel_size,
+            std::array<uint32_t, 2> stride,
+            std::array<uint32_t, 2> padding,
+            std::array<uint32_t, 2> dilation,
+            uint32_t groups,
+            ttnn::MeshDevice *device,
+            std::optional<const Conv2dConfig> conv_config_) -> ttnn::Tensor {
+                return ttnn::operations::conv::conv2d::prepare_conv_bias(
+                    bias_tensor, input_memory_config, in_channels, out_channels, batch_size, input_height, input_width, kernel_size, stride, padding, dilation, groups, device, conv_config_);
+        },
```

Comment:
you know that you don't need to do this if what you expect python args to be is 1-1 to c++?